src/tentacles/core.clj

```diff
@@ -106,9 +106,15 @@
                         (when otp
                           {:headers {"X-GitHub-OTP" otp}})
                         (when if-modified-since
-                          {:headers {"if-Modified-Since" if-modified-since}}))
+                          {:headers {"if-Modified-Since" if-modified-since}})
+                        (when conn-timeout
+                          {:conn-timeout conn-timeout})
```

Comment:
:information_source: only: We're using `clj-http` 3.1.0 in `tentacles`, but in latest stable (3.10.0), `conn-timeout` & `conn-request-timeout` are deprecated in favour of `connect-*`.

(I only found this because I'm thoroughly unfamiliar with this code so I was checking everything back!)

* https://github.com/dakrone/clj-http/commit/1e77ba2c73b15534522c18eed6e159fc6ef3d977, https://github.com/dakrone/clj-http/blob/3.x/changelog.org#3100

Reply:
We're using version 2.3.0 in circleci/circle: https://github.com/circleci/circle/blob/master/project.clj#L31 so we should probably test this change against that version as well.

Reply:
yep, 2.3 in circle, `conn-*` works there